libs/sdk-ui-dashboard/src/model/commands/filters.ts

```diff
@@ -339,16 +394,27 @@ export interface ChangeAttributeFilterSelection extends IDashboardCommand {
  * The attribute elements can be provided either using their URI (primary key) or value. Together with the
  * elements you must indicate the selection type - either 'IN' or 'NOT_IN'.
  *
+ * To convert {@link IDashboardFilter} to {@link @gooddata/sdk-model#IFilter} use {@link dashboardAttributeFilterToAttributeFilter}.
+ * Converted filter can be used within the command's payload.
+ *
  * @remarks see {@link resetAttributeFilterSelection} for convenience function to select all attribute elements of
  *  particular filter.
  *
+ *  For determination of what selection type to use, use {@link @gooddata/sdk-model#isPositiveAttributeFilter} function.
```

Comment:
Maybe instead suggest using the applyAttributeFilter? That has much easier conversion etc.